SingularityUI/gulpfile.js

```diff
@@ -50,7 +50,8 @@ var WebpackDevServer = require('webpack-dev-server');
 gulp.task("clean", function() {
   return del([
     path.resolve(dest, 'static/**'),
-    path.resolve(dest, 'index.html')], {force: true});
+    path.resolve(dest, 'index.html'),
+    dest], {force: true});
```

Comment:
i had to add `{force: true}` in the original implementation because we were deleting a folder outside the working directory -- this shouldnt be needed anymore